Source/Atomic/Resource/Image.cpp

```diff
@@ -246,7 +247,9 @@ Image::Image(Context* context) :
     components_(0),
     cubemap_(false),
     array_(false),
-    sRGB_(false)
+    sRGB_(false),
+    pot_(false),
+    hasAlphaChannel_(false)
 {

```

Comment:
Sure, I can move both to be just the accessors, they're only using already cached data so that should be fine.